project-base/docker/php-fpm/Dockerfile

```diff
@@ -31,12 +31,17 @@ RUN chmod +x /usr/local/bin/docker-install-composer && \
 # autoconf needed by "redis" extension
 RUN apt-get install -y \
     libpng-dev \
+    libjpeg-dev \
+    libfreetype6-dev \
```

Comment:
do we really need the free type library?

Reply:
or... why do we need the free type library?

Reply:
i do not know. it was just copied from alpine version that we had before.

Reply:
https://github.com/shopsys/shopsys/issues/741